stellar-dotnet-sdk/Transaction.cs

```diff
@@ -6,28 +6,25 @@
 
 namespace stellar_dotnet_sdk
 {
-    public class Transaction
+    public class Transaction : TransactionBase
     {
-        private const uint BaseFee = 100;
-
-        private Transaction(KeyPair sourceAccount, uint fee, long sequenceNumber, Operation[] operations, Memo memo, TimeBounds timeBounds)
+        public Transaction(IAccountId sourceAccount, uint fee, long sequenceNumber, Operation[] operations, Memo memo, TimeBounds timeBounds)
```

Comment:
if you want users of the SDK to only be able to create `Transaction` instances using `TransactionBuilder` I think you can do that by making the constructor internal.

Reply:
That would be a breaking change. Let's see what @elucidsoft says about it.

Reply:
We are trying to avoid breaking changes for this release. So we will probably not do this at this time.  Can should add a deprecate attribute to warn of future deprecation.